src/app/wallet/wallet-tx-history-item/wallet-tx-history-item.component.html

```diff
@@ -1,8 +1,8 @@
 <ng-container *ngIf="(txView$ | async) as tx">
-  <div class="flex gap-2">
+  <div class="flex items-center gap-2">
```

Comment:
The arrow icon was not centered vertically.
Before:
![Screenshot 2021-09-22 at 17 37 06](https://user-images.githubusercontent.com/12549625/134380948-c4f526c4-d9fc-4d0b-81ce-76921444921a.png)
After:
![Screenshot 2021-09-22 at 17 38 17](https://user-images.githubusercontent.com/12549625/134380953-853afc4a-3be9-4735-a640-8b101cfae107.png)